api/contract.go

```diff
@@ -28,8 +28,8 @@ func (r *createFilterRequest) Valid() error {
 		return errors.New("missing field: dataset")
 	}
 
-	if r.Dataset.ID == "" || r.Dataset.Edition == "" || r.Dataset.Version == 0 {
-		return errors.New("missing field: [dataset.id | dataset.edition | dataset.version]")
+	if r.Dataset.ID == "" || r.Dataset.Edition == "" || r.Dataset.Version == 0 || r.Dataset.LowestGeography == "" {
+		return errors.New("missing field: [dataset.id | dataset.edition | dataset.version | dataset.lowest_geography]")
```

Comment:
I don't think this is right.. we shouldn't need to rely on the user to specify the lowest_geography, it should be saved to the dataset for us to retrieve when we call GetVersion. It's been added to the instance in dataset-api and dp-api-clients-go in my PRs.